src/fileprovider.cs

```diff
@@ -488,6 +488,9 @@ interface NSFileProviderItem {
 		[Export ("favoriteRank", ArgumentSemantic.Copy)]
 		NSNumber GetFavoriteRank ();
 
+#if XAMCORE_4_0 // Not available in mac
+		[NoMac]
+#endif
```

Comment:
let's add an `[Obsolete]` message (on macOS) so people won't think it should work
*and* won't be surprised if the member is removed later (at `XAMCORE_4_0` time)

Reply:
Good call 👍

Reply:
Fixed, thanks @spouliot